config/routes/candidate.rb

```diff
@@ -335,45 +335,45 @@
       delete '/delete/:id' => 'degrees/destroy#destroy'
     end
 
-    scope '/continuous-applications' do
-      get '/choose' => 'course_choices/do_you_know_which_course#new', as: :continuous_applications_do_you_know_the_course
+    scope '/course-choices' do
```

Comment:
What's your method to rename these routes? It would be nice to include that in the PR description

Reply:
Great question 💭 most of the approach is outlined by the separate commits. The first commit changed the `scope` and any related tests etc. The rest of the changes were largely find and replace based on the `as:` arguments.
Running the entire test suite for each commit was too time consuming, so I'd run sections of the suite and then at the end of a day or lunchtime run the entire suite for a collection of changes.

This was a different approach to the Controller changes, where I could use my editor to rename a Controller Class

Reply:
That's a great approach!